kitsune/flagit/views.py

```diff
@@ -69,6 +71,13 @@ def update(request, flagged_object_id):
     flagged = get_object_or_404(FlaggedObject, pk=flagged_object_id)
     new_status = request.POST.get("status")
     if new_status:
+        ct = flagged.content_type
+        # If the object is an Answer let's fire a notification
+        # if the flag is invalid
+        if new_status == str(FlaggedObject.FLAG_REJECTED) and ct.model_class() == Answer:
```

Comment:
Will `new_status` always be a string? Might it be better to do `str(new_status) == str(FlaggedObject.FLAG_REJECTED)`?